README.md

```diff
@@ -6,7 +6,9 @@
 [![Discord](https://img.shields.io/discord/930865462852591648?label=Discord&logo=Discord&logoColor=white)](https://discord.gg/pzvAKPKyHM)
 
 <p align="center">
-    <img src="src/public/chaiNNer screenshot.png" width="720" />
+  <a href="https://github.com/joeyballentine/chaiNNer/releases" target="_blank">
+    <img src="src/public/banner.png" width="720" />
```

Comment:
I don't think the text is visible enough on a white background.

![image](https://user-images.githubusercontent.com/20878432/182024176-942329d2-2a6e-46c7-8643-1b30d691baf1.png)